assets/base.css

```diff
@@ -3048,3 +3048,14 @@ details-disclosure > details {
     outline: transparent solid 1px;
   }
 }
+
+/* Fixed background */
+
+.animate--fixed {
```

Comment:
Regarding the bug you're seeing on Chrome, I tried [this solution](https://stackoverflow.com/questions/70296340/problems-with-css-clip-path-in-google-chrome) and it seemed to work for me. It's worth giving it a try and testing it more extensively.

Reply:
Ah yes, thanks for finding that. I thought my days of using the `translateZ(0)` hack were over, but it appears to still be relevant! 😅 I don't love kicking the GPU acceleration unnecessarily, but it may be our best option for now. 

I pushed this in a64ea39 for now, but used `-webkit-transform` instead of `transform` (so that it won't trigger unnecessarily for FF), and I also added a code comment explaining it.

Reply:
😩 Oh whoops. You're correct. We'll need to re-evaluate this.